### ISSUE TITLE
Wlan Portal Template throws error when no logo is provided

### ISSUE THREAD

**Issue by zbuchheit** [Juniper/terraform-provider-mist#32]
When I don't provide a logo to my wlan_portal_template I receive an error 
```
│ Error: Invalid "logo" value for "mist_site_wlan_portal_template" resource
│ 
│   with mist_site_wlan_portal_template.wlan_one,
│   on main.tf line 22, in resource "mist_site_wlan_portal_template" "wlan_one":
│   22: resource "mist_site_wlan_portal_template" "wlan_one" {
│ 
│ Unable to open the file "": open : no such file or directory
```

## Example

```tf
resource "mist_site_wlan_portal_template" "wlan_one" {
  site_id = "redacted"
  wlan_id = mist_site_wlan.wlan_one.id
  portal_template = {
    sms_message_format    = "Code {{code}} expires in {{duration}} minutes."
    sms_validity_duration = "10"
    page_title            = "Welcome To My Demo Portal"
    locales = {
      "fr-FR" = {
        page_title = "Bienvenue sur mon portail de démo"
      }
    }
    # logo = "../pulumi_logo.png" Throws error
  }
}
```

**Comment by tmunzer** [comment]
Hi @zbuchheit , it should be fixed with v0.1.35

**Comment by zbuchheit** [comment]
confirming it was fixed. thanks!
